### PR TITLE
TypeDefination fix for selenoid options

### PR DESCRIPTION
**Pull request by suniljaiswal01**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)
Update Type Definition For Selenoid Option

https://github.com/webdriverio/webdriverio/issues/5675

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/webdriver/webdriver.d.ts [hunk 1]
```diff
@@ -351,6 +351,12 @@ declare namespace WebDriver {
             [name: string]: any;
         };
 
+        //selenoid specific
+        "selenoid:options"?: {
+            enable: boolean,
+            enableVideo: boolean
```
Comment (rytikovroman1994) [inline packages/webdriver/webdriver.d.ts]: need all option
// seleoid specific
        "selenoid:options"?: { 
            enableVNC?: boolean,  
            enableVideo?: boolean, 
            videoName?: string,
            screenResolution?: string,
            videoScreenSize?: string,
            videoFrameRate?: number,
            videoCodec?: string
        },

**Comment by christian-bromann** [comment]
Looks good so far, could we add a typings test in this file: https://github.com/webdriverio/webdriverio/blob/master/tests/typings/webdriverio/config.ts to check that WebdriverIO understands selenoid capabilities?

**Comment by rytikovroman1994** [comment]
I checked locally. Everything works perfectly.

**Comment by BorisOsipov** [comment]
@suniljaiswal01 could you add all special capabilities? https://aerokube.com/selenoid/latest/#_special_capabilities

**Comment by suniljaiswal01** [comment]
> Looks good so far, could we add a typings test in this file: https://github.com/webdriverio/webdriverio/blob/master/tests/typings/webdriverio/config.ts to check that WebdriverIO understands selenoid capabilities?

Test Added

**Comment by suniljaiswal01** [comment]
> @suniljaiswal01 could you add all special capabilities? https://aerokube.com/selenoid/latest/#_special_capabilities

All the capabilities added

**Pull request merged**